api/v1beta2/mysqlcluster_webhook.go

```diff
@@ -0,0 +1,11 @@
+package v1beta2
+
+import (
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+func (r *MySQLCluster) SetupWebhookWithManager(mgr ctrl.Manager) error {
+	return ctrl.NewWebhookManagedBy(mgr).
+		For(r).
+		Complete()
+}
```

Comment:
I think the defaulting and validating webhooks should be implemented for `v1beta2`.

Reply:
I thought this was unnecessary in v1beta2, because if conversion is working, the validating/mutating webhook defined in v1beta1 will do the job.

Reply:
@d-kuro 
Have you experimented with that area?
That's only true when the webhook configuration `matchPolicy` is set to `true`.
https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#matching-requests-matchpolicy

I'd strongly suggest implementing the webhook for v1beta2 because v1beta1 will soon be removed.

Reply:
Thanks for the helpful links!
I checked the operation locally, but I didn't have enough knowledge that it wasn't because the conversion was running 🙏 

As pointed out in another comment, I'm going to move the conversion code to v1beta1, so I will implement it by the way.

Reply:
I added AdmissionWebhook to v1beta2 API.
https://github.com/cybozu-go/moco/pull/350/commits/42da61056edfa67d7a91dfc15129c0ad3d306e7c